pkg/options/options.go

```diff
@@ -57,8 +67,126 @@ type GCPAPIKey struct {
 }
 
 type AzureAPIKey struct {
-	BaseDnsDomain        string `yaml:"baseDnsDomain"`
-	BaseDomainRGN        string `yaml:"azureBaseDomainRGN"`
-	ServicePrincipalJson string `yaml:"azureServicePrincipalJson"`
-	Region               string `yaml:"region"`
+	BaseDnsDomain  string `yaml:"baseDnsDomain"`
+	BaseDomainRGN  string `yaml:"azureBaseDomainRGN"`
+	ClientId       string `yaml:"clientId"`
+	ClientSecret   string `yaml:"clientSecret"`
+	TenantId       string `yaml:"tenantId"`
+	SubscriptionId string `yaml:"subscriptionId"`
+	Region         string `yaml:"region"`
+}
+
+const charset = "abcdefghijklmnopqrstuvwxyz" +
+	"0123456789"
+
+var TestOptions TestOptionsT
+
+func LoadOptions(optionsFile string) error {
+	err := unmarshal(optionsFile)
+	if err != nil {
```

Comment:
can combine these 2 statements

Reply:
yes we can... it is only cosmetic.

Reply:
No we can't because we want to return the error

Reply:
oh got it